HKMP/Api/Client/IClientManager.cs

```diff
@@ -90,4 +90,14 @@ public interface IClientManager {
     /// Event that is called when another player leaves the local scene.
     /// </summary>
     event Action<IClientPlayer> PlayerLeaveSceneEvent;
+
+    /// <summary>
+    /// Event that is called when HKMP modifies the game's time scale.
+    /// </summary>
+    delegate void SetTimeScale(float timeScale);
+
+    /// <summary>
+    /// Event that is called when HKMP modifies the game's time scale.
+    /// </summary>
+    event SetTimeScale OnSetTimeScale;
```

Comment:
Please see the other events in this class and their naming. I prefer not to use `On...` for event names, since that is more suited for methods that subscribe to such an event.